app/routes.js

```diff
@@ -111,8 +108,7 @@ router.get('/form-designer/edit-page/:pageId', function (req, res) {
       pageIndex: pageIndex,
       pageData: pageData,
       editingExistingQuestion: req.session.data.pages[pageIndex] !== undefined,
-      enableMultipleChoiceAnswerType,
-      enableNumberAnswerType
```

Comment:
There's a reference to this in [README.MD](https://github.com/alphagov/forms-prototypes/blob/main/README.md) which can be removed as well

Reply:
Good spot, will do